### PR TITLE
Add unit tests for ConcurrencyThrottlingUtils class

### PR DESCRIPTION
**Pull request by GDLMadushanka**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+package org.apache.synapse.util;
+
+import junit.framework.TestCase;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.SynapseConstants;
+import org.apache.synapse.commons.throttle.core.ConcurrentAccessController;
+import org.apache.synapse.mediators.TestUtils;
+import org.junit.Assert;
+
+public class ConcurrencyThrottlingUtilsTest extends TestCase {
```
Comment (riyafa) [inline modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java]: Shall we add class javadoc?

Comment (vinok88) [inline modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java]: Add a Doc comment explaining what is been tested here.

File: modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    public void testDecrement() throws Exception {
```
Comment (riyafa) [inline modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java]: Also test description?

File: modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java (hunk 1 above)
Comment (GDLMadushanka) [reply modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java]: +1 Done

File: modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java (hunk 2 above)
Comment (GDLMadushanka) [reply modules/core/src/test/java/org/apache/synapse/util/ConcurrencyThrottlingUtilsTest.java]: +1 Done

**Pull request merged**